spec/index.bs

```diff
@@ -7165,6 +7165,11 @@ GPUQueue includes GPUObjectBase;
                     <div class=validusage>
                         - Every {{GPUBuffer}} referenced in any element of |commandBuffers| is in the
                             `"unmapped"` [=buffer state=].
+                        - Every {{GPUQuerySet}} referenced in any element of |commandBuffers| is in the
+                            `"available"` [=query set state=]. Unlike the other types of query set,
+                            {{GPURenderPassDescriptor}}.{{GPURenderPassDescriptor/occlusionQuerySet}}
+                            is not referenced at {{GPURenderPassEncoder}} creation, but when
+                            {{GPURenderPassEncoder/beginOcclusionQuery()}} is called.
```

Comment:
> Unlike the other types of query set

This isn't actually difference from the other types, is it? What's special here is not the type, but the field `GPURenderPassDescriptor.occlusionQuerySet`, which is already unique to occlusion queries.

I would write: `Every {{GPUQuerySet}} referenced in a command in any element of |commandBuffers| is in the [=query set state/"available"=] state. For occlusion queries, {{GPURenderPassDescriptor/occlusionQuerySet}} in {{GPUCommandEncoder/beginRenderPass()}} does not constitute a reference, while {{GPURenderPassEncoder/beginOcclusionQuery()}} does`